### PR TITLE
Sass and bem check ts

### PR DESCRIPTION
**Pull request by tauri-st**
Sass nesting and BEM updates. Everything is pretty self explanatory from the commits,

The biggest change I made was to put the recipe-examples__desc into the search results container bc it references the recipe cards. This required me to move the "hidden" class from the search results container to the straight search results bc the desc and the results have different times when they need to hide and unhide. I changed all of the references to hide and unhide the searchResults to the recipeList instead in the JS. I removed the searchResults variable but included a comment to reference how recipeList is also an element to hide an unhide in addition to how it is declared above as a dom element to listen to and receive data

### CODE REVIEW

File: docs/css/main.scss [hunk 1]
```diff
@@ -83,20 +92,20 @@ header {
     .quote-section {
```
Comment (terrifricker) [inline docs/css/main.scss]:
```suggestion
    .quote-and-suggestions-section {
```

**Review by terrifricker — CHANGES_REQUESTED** [review]
Just one missed "quotes-and-suggestions-section"

**Comment by tauri-st** [comment]
Further:
- Change recipeList variable in JS to searchResults instead and change all references to it within JS
- Move CSS for recipe-examples__desc into search container section and nest this + recipes styles into the search-container

**Review by terrifricker — APPROVED** [review]
Such a great pull request @tauri-st!  You made some much needed changes.

**Pull request merged**